packages/checkout/widgets-lib/src/widgets/connect/components/WalletList.tsx

```diff
@@ -316,6 +333,20 @@ export function WalletList(props: WalletListProps) {
         }}
         onWalletChange={handleWalletChange}
       />
+
+      <ChangedYourMindDrawer
+        visible={showChangedYourMindDrawer}
+        checkout={checkout!}
+        onCloseDrawer={() => setShowChangedYourMindDrawer(false)}
+        onTryAgain={onChosenProviderDetailChange}
+      />
+
+      <UnableToConnectDrawer
+        visible={showUnableToConnectDrawer}
+        checkout={checkout!}
+        onCloseDrawer={() => setShowUnableToConnectDrawer(false)}
+        onTryAgain={() => setShowUnableToConnectDrawer(false)}
```

Comment:
Would like your opinion around this - do you think when there is an unknown/other error than user rejected error, do we want try again button to just close the drawer? 

I'm just thinking because we might not know what the error is, maybe we want to restart the whole wallet selection flow rather than keep trying to call the same code that retries a specific wallet connection flow.

If this is the behaviour we want (close drawer), I can make changes to the drawer component itself so that we don't have to pass in the same thing for `onCloseDrawer` and `onTryAgain`.

Reply:
But also might need to consider the copy of the Unable to connect drawer - currently it says: `Check your wallet and try again, or close and try a different option`.

Reply:
yeah I was thinking if there was an unknown error we would bomb out to the whole error view screen. But might be hard to tell the variety of errors we get here..